### PR TITLE
Fix typos in .md files

### PR DESCRIPTION
**Pull request by tocic**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -14,6 +14,6 @@ Read the [Contribution Guidelines](https://github.com/nlohmann/json/blob/develop
...
+- Specifically, I am aware of compilation problems with **Microsoft Visual Studio** (there even is an [issue label](https://github.com/nlohmann/json/issues?utf8=✓&q=label%3A%22visual+studio%22+) for this kind of bugs). I understand that even in 2016, complete C++11 support isn't there yet. But please also understand that I do not want to drop features or uglify the code just to make Microsoft's sub-standard compiler happy. The past has shown that there are ways to express the functionality such that the code compiles with the most recent MSVC - unfortunately, this is not the main objective of the project.
```
Comment (falbrechtskirchinger) [inline .github/PULL_REQUEST_TEMPLATE.md]: That's still not correct.

```suggestion
- Specifically, I am aware of compilation problems with **Microsoft Visual Studio** (there even is an [issue label](https://github.com/nlohmann/json/issues?utf8=✓&q=label%3A%22visual+studio%22+) for this kind of bug). I understand that even in 2016, complete C++11 support isn't there yet. But please also understand that I do not want to drop features or uglify the code just to make Microsoft's sub-standard compiler happy. The past has shown that there are ways to express the functionality such that the code compiles with the most recent MSVC - unfortunately, this is not the main objective of the project.
```

File: docs/mkdocs/docs/api/basic_json/update.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ function.
 
 ??? example
 
-    One common usecase for this function is the handling of user settings. Assume your application can configured in
+    One common usecase for this function is the handling of user settings. Assume your application can be configured in
```
Comment (falbrechtskirchinger) [inline docs/mkdocs/docs/api/basic_json/update.md]:
```suggestion
    One common use case for this function is the handling of user settings. Assume your application can be configured in
```

File: docs/mkdocs/docs/api/macros/nlohmann_json_serialize_enum.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ The `NLOHMANN_JSON_SERIALIZE_ENUM` allows to define a user-defined serialization
 :   name of the enum to serialize/deserialize
 
 `conversion` (in)
-:   a pair of an enumerator and a JSON serialization; arbitrary pairs can can be given as comma-separated list
+:   a pair of an enumerator and a JSON serialization; arbitrary pairs can be given as comma-separated list
```
Comment (falbrechtskirchinger) [inline docs/mkdocs/docs/api/macros/nlohmann_json_serialize_enum.md]:
```suggestion
:   a pair of an enumerator and a JSON serialization; arbitrary pairs can be given as a comma-separated list
```

**Review by falbrechtskirchinger — CHANGES_REQUESTED** [review]

**Review by falbrechtskirchinger — APPROVED** [review]
Looks good to me.

**Comment by nlohmann** [comment]
Thanks!

**Pull request merged**